PySortedInputToolExample/PySortedInputToolExampleEngine.py

```diff
@@ -0,0 +1,206 @@
+        :param b_has_errors: Boolean; set to true to not do the final processing.
+        :return: Void
+        """
+        pass
+
+    def ii_init(self, record_info_in):
+        """
+        Handles type validation, and renames the outgoing field.
+        Called when the incoming connection's record metadata is available or has changed, and
+        has let the Alteryx engine know what its output will look like.
+        :param record_info_in: A RecordInfo object containing the XML representation for the incoming connection's field and sort properties.
+        :return: False if type validation fails.
+        """
+        #  Evaluate field type of selected field to make sure only numeric field types are selected
+        if 'String' in str(self.get_single_field_type(record_info_in))\
+                or 'Bool' in str(self.get_single_field_type(record_info_in)):
+            self.output_message('pi_push_all_records', Sdk.EngineMessageType.error, 'Selected A Non-Numeric Field')
+            return False
+        else:
+            self.record_info_in = record_info_in
+            self.record_info_out = self.record_info_in
+            # Rename outgoing field
+            self.record_info_out.rename_field_by_name(
+                self.field_selection,
+                '{0}_{1}'.format(self.selected_aggregation, self.field_selection)
+            )
+            # Copy over record_info_in to record_info_out
+            self.output_anchor.init(self.record_info_out)
+            self.record_creator = self.record_info_out.construct_record_creator()
+            self.record_copier = Sdk.RecordCopier(self.record_info_out, self.record_info_in)
+            # Map each column of the input to where we want in the output
+            for idx in range(len(self.record_info_in)):
+                self.record_copier.add(idx, idx)
+            self.record_copier.done_adding()
+            # Let ii_put_record know that ii_init has been initialized
+            self.initialized = True
+        return True
+
+    def ii_push_record(self, in_record):
+        """
+        Responsible for pushing the first record out - either the highest or lowest numeric value of the field.
+        Called when an input record is being sent to the plugin.
+        :param in_record: The data for the incoming record.
+        :return: Will return False if:
+          1. ii_init has not been initialized
+          2. ii_push_record calling limit has been reached.
+          3. If record_cnt > 1
+        """
+        if not self.initialized:
+            return False
+        self.record_cnt += 1
+        if self.record_cnt == 1:
+            self.record_creator.reset()
+            self.record_copier.copy(self.record_creator, in_record)
+            out_record = self.record_creator.finalize_record()
+            # Push the record and quit if there's a downstream error
+            if not self.output_anchor.push_record(out_record):
```

Comment:
Actually, if you're only pushing the first record down, try just always `return False`